spk/owncloud/src/bin/owncloud-daemon

```diff
@@ -7,26 +7,24 @@ WEB_DIR="/var/services/web_packages"
 # for backwards compatability
 if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ];then
     WEB_DIR="/var/services/web"
+	if [ -z ${SYNOPKG_PKGDEST} ]; then
+		SYNOPKG_PKGDEST="/var/packages/owncloud/target"
+	fi
```

Comment:
SYNOPKG_PKGDEST is always defined, when the service is started by synopkg (as SYNOPKG_DSM_VERSION_MAJOR is).

Reply:
I've removed it from the daemon, should it be removed from all the wizard files? I found that sometimes in the interface parameters like SYNOPKG_PKGDEST_VOL weren't always known so I added all of these fail-safes.

Reply:
Regarding the environment variables available to the UI wizard I realize they are not the same depending on the platform and stage. So I created a generic UI wizard script to extract the various variables (https://pastebin.com/xDdbTYLV). I was them able to collate the results as follows:

DSM 6.2: https://pastebin.com/n3e5jbRr
DSM 7.1: https://pastebin.com/7nKEC4eN

I believe this can be a useful resource for other devs to know what variables are and are not available when creating UI wizards. I'll audit the variables in use against this and prune what doesn't need to be there.